### PR TITLE
Unify help description text styling

### PR DESCRIPTION
**Pull request by karmatosed**
This brings in the toolbar styling across help classes and also to the base help class. In my work on options I am currently having to work around the italic styling, so this will avoid me having to do that by fixing the base style. It also brings in the style to the sidebar. Linked to issue #25851.

## Visuals of PR in action
image

## Feedback
I am generally looking for code review on this PR and also looking for design feedback, specifically around where this could also be iterated if anything was missed class wise or other aspects.

### CODE REVIEW

**Comment by jasmussen** [comment]
Before:

Screenshot 2020-10-07 at 10 36 09
Screenshot 2020-10-07 at 10 36 29

After:

Screenshot 2020-10-07 at 10 36 43
Screenshot 2020-10-07 at 10 36 51

This seems more legible to me than the italic font. I'm a fan.

File: packages/components/src/base-control/style.scss [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 	.components-base-control__help {
 		margin-top: -$grid-unit-10;
-		font-style: italic;
+		font-size: $default-font-size - 1px;
```
Comment (jasmussen) [inline packages/components/src/base-control/style.scss:20]: I'd create a new $helptext-font-size and reference that directly.

Comment (karmatosed) [reply packages/components/src/base-control/style.scss:20]: Oh, that's a great idea, spinning up those changes now! Thanks for review.

File: packages/components/src/form-token-field/style.scss [hunk 2]
```diff
@@ -48,7 +48,9 @@
 }
 
 .components-form-token-field__help {
-	font-style: italic;
+	font-size: $default-font-size - 1px;
```
Comment (aaroncampbell) [inline packages/components/src/form-token-field/style.scss:51]: Should this be using the new `$helptext-font-size` as well?

Comment (karmatosed) [reply packages/components/src/form-token-field/style.scss:51]: Ah yes! I think I also found another instance. @aaroncampbell would you be able to do another code review, please? Fingers crossed I've caught all those pesky variables now. Thanks so much.

**Review by aaroncampbell — APPROVED** [review]
Checked it out and looked for any other occurrences of `$default-font-size` being adjusted and didn't see any. Looks good to me.

**Comment by MichaelArestad** [comment]
I almost wonder if there should be a `<HelpText>` component with a few props for flexibility.

**Pull request merged**